projects/laji-ui/src/lib/checkbox/checkbox.component.scss

```diff
@@ -29,14 +29,14 @@ input {
     0 0 1px 1px rgba($neutral-7, 0.4),
     inset 1px 2px 4px 0 rgba($neutral-7, 0.1);
   border-radius: 1px;
-  &:focus {
-    outline: unset;
-  }
   &:checked::after {
     position: absolute;
     left: 0.15em;
     top: -0.15em;
     content: "✓";
     color: $neutral-7;
   }
+  &:focus:not(:focus-visible) {
+    outline: none;
+  }
```

Comment:
Even `:not` is not supported by IE11 :)

The decision to drop IE 11 support was made on Friday so these shouldn't be problem anymore, but instead of having each individual component to decide what to do with focus outline I think this should be made a global rule.